.test-infra/jenkins/NexmarkBuilder.groovy

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import CommonJobProperties as commonJobProperties
+
+// Class for building nexmark jobs and suites.
+class NexmarkBuilder {
+
+  private static Map<String, Object> defaultOptions = [
+          'bigQueryTable'          : 'nexmark',
```

Comment:
Can we make this different for the phrase triggered jobs vs the automatic jobs so we don't have random PRs in the perf data from master?

Reply:
Nevermind, looks like you did it on dataset instead of table. That will be fine too.

Reply:
Yes, I changed the dataset instead. Table names are also constructed by some nexmark utils, so it is better to have them named the same in case naming convention changes (easier to compare if needed).